libr/core/hack.c

```diff
@@ -253,7 +253,7 @@ R_API int r_core_hack(RCore *core, const char *op) {
 	}
 	if (hack) {
 		RAnalOp analop;
-		if (!r_anal_op (core->anal, &analop, core->offset, core->block, core->blocksize, R_ANAL_OP_MASK_ALL)) {
```

Comment:
any benchmark to show the benefit of this change?

Reply:
i don't have atm , i shall try to produce one , but more than about optimization , this thing at least reduces lot of allocation done to fill op->esil  , where it's not needed in most of the cases